default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	"files.autoSaveDelay": 1000,
+
+	// The default language identifier that is assigned to new files. If configured to `${activeEditorLanguage}`, will use the language identifier of the currently active text editor if any.
+	"files.defaultLanguage": "",
+
+	// Moves files/folders to the OS trash (recycle bin on Windows) when deleting. Disabling this will delete files/folders permanently.
+	"files.enableTrash": true,
+
+	// The default character set encoding to use when reading and writing files. This setting can also be configured per language.
+	//  - utf8: UTF-8
+	//  - utf8bom: UTF-8 with BOM
+	//  - utf16le: UTF-16 LE
+	//  - utf16be: UTF-16 BE
+	//  - windows1252: Western (Windows 1252)
+	//  - iso88591: Western (ISO 8859-1)
+	//  - iso88593: Western (ISO 8859-3)
+	//  - iso885915: Western (ISO 8859-15)
+	//  - macroman: Western (Mac Roman)
+	//  - cp437: DOS (CP 437)
+	//  - windows1256: Arabic (Windows 1256)
+	//  - iso88596: Arabic (ISO 8859-6)
+	//  - windows1257: Baltic (Windows 1257)
+	//  - iso88594: Baltic (ISO 8859-4)
+	//  - iso885914: Celtic (ISO 8859-14)
+	//  - windows1250: Central European (Windows 1250)
+	//  - iso88592: Central European (ISO 8859-2)
+	//  - cp852: Central European (CP 852)
+	//  - windows1251: Cyrillic (Windows 1251)
+	//  - cp866: Cyrillic (CP 866)
+	//  - iso88595: Cyrillic (ISO 8859-5)
+	//  - koi8r: Cyrillic (KOI8-R)
+	//  - koi8u: Cyrillic (KOI8-U)
+	//  - iso885913: Estonian (ISO 8859-13)
+	//  - windows1253: Greek (Windows 1253)
+	//  - iso88597: Greek (ISO 8859-7)
+	//  - windows1255: Hebrew (Windows 1255)
+	//  - iso88598: Hebrew (ISO 8859-8)
+	//  - iso885910: Nordic (ISO 8859-10)
+	//  - iso885916: Romanian (ISO 8859-16)
+	//  - windows1254: Turkish (Windows 1254)
+	//  - iso88599: Turkish (ISO 8859-9)
+	//  - windows1258: Vietnamese (Windows 1258)
+	//  - gbk: Simplified Chinese (GBK)
+	//  - gb18030: Simplified Chinese (GB18030)
+	//  - cp950: Traditional Chinese (Big5)
+	//  - big5hkscs: Traditional Chinese (Big5-HKSCS)
+	//  - shiftjis: Japanese (Shift JIS)
+	//  - eucjp: Japanese (EUC-JP)
+	//  - euckr: Korean (EUC-KR)
+	//  - windows874: Thai (Windows 874)
+	//  - iso885911: Latin/Thai (ISO 8859-11)
+	//  - koi8ru: Cyrillic (KOI8-RU)
+	//  - koi8t: Tajik (KOI8-T)
+	//  - gb2312: Simplified Chinese (GB 2312)
+	//  - cp865: Nordic DOS (CP 865)
+	//  - cp850: Western European DOS (CP 850)
+	"files.encoding": "utf8",
+
+	// The default end of line character.
+	//  - \n: LF
+	//  - \r\n: CRLF
+	//  - auto: Uses operating system specific end of line character.
+	"files.eol": "auto",
+
+	// Configure [glob patterns](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options) for excluding files and folders. For example, the file explorer decides which files and folders to show or hide based on this setting. Refer to the `search.exclude` setting to define search-specific excludes.
+	"files.exclude": {
+		"**/.git": true,
```

Comment:
Hide `.vscode` folder?

Reply:
Interesting! I never knew this was a setting.